docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
       - 15432:5432
     volumes:
       - .:/root/go/src/github.com/percona/pmm-managed
+      - ../pmm:/root/go/src/github.com/percona/pmm
```

Comment:
let's revert it, it might bring new problems.

Reply:
I reverted it